server/service/appconfig.go

```diff
@@ -101,8 +102,10 @@ func getAppConfigEndpoint(ctx context.Context, request interface{}, svc fleet.Se
 }
 
 func (svc *Service) AppConfig(ctx context.Context) (*fleet.AppConfig, error) {
```

Comment:
An alternative is to leave `Service.AppConfig` untouched, and introduce a new `Service.GetOrigLogoURL`.

Reply:
I'd be concerned that wouldn't age well if it turns out we need a couple more fields from app config. And in any case, the service would still need to authorize access in a similar manner.